docs/articles/documentation/test-api/A-Z.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: A-Z
+permalink: /documentation/test-api/a-z.html
+---
+# A-Z
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
+
+* [afterEach](test-code-structure.html#initialization-and-clean-up)
+* [beforeEach](test-code-structure.html#initialization-and-clean-up)
+* [fixture](test-code-structure.html#fixtures)
+* [page](test-code-structure.html#specifying-the-start-webpage)
+* [test](test-code-structure.html#tests)
+
+## Objects with Members
+
+* [ClientFunction](obtaining-data-from-the-client.html#creating-client-functions)
+    * [with](obtaining-data-from-the-client.html#overwriting-options)
+* [DOM Node Snapshot (with members)](selecting-page-elements/dom-node-snapshots.html)
```

Comment:
Again, `with members` is not quite clear.
Maybe *members* in italics as a single child item?

```
* [DOM Node Snapshot](selecting-page-elements/dom-node-snapshots.html)
    * *[members](selecting-page-elements/dom-node-snapshots.html)*
```